runtime/astar/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "astar-runtime"
-version = "5.45.0"
+version = "5.46.0"
```

Comment:
I think we can remove versioning from runtime now

Reply:
Because we don't need it. How do we bump it anyway

Reply:
Right now it looks like it's connected to client release

Reply:
The versions are still "aligned", that's true, but they are not connected.

The bump is in respect to the crate level, same as until now I guess?
Breaking changes or new major updates would entail major version bump, otherwise minor or patch where appropriate.

Just my opinion - but wouldn't it be weird to release new versions of a "crate", while keeping the crate version same?

Reply:
No, we don't release it in a sense of publishing it, but it's still built into the binary and version is reflected in the .lock file. My comment was about that.

But no strong opinion here - if team agrees, let's go with it.

Reply:
Or we can align it with spec. 1.10.0

Reply:
To be fair, it would have to be `1.11.0` then 🙂. The last two digits are patch version, the first two (or more) are a combination of major/minor version.

Reply:
I'm in, we can use `1.11.0`

Reply:
Actually, I meant to write `11.0.0` 🤦 . But let's keep in mind that we're on `5.x.y` now, and going back to `1` as major version would be weird.

Reply:
Given that our runtime release format is **XXYY**, what would the version be for a minor runtime release like **1101**? Would it be `11.1.0`? I assume the patch digit would always remain 0 in that case